container/kvm/libvirt/domainxml.go

```diff
@@ -280,7 +277,8 @@ type GIC struct {
 type CPU struct {
 	Mode  string `xml:"mode,attr,omitempty"`
 	Match string `xml:"match,attr,omitempty"`
-	Model Model  `xml:"model,omitempty"`
+	Check string `xml:"check,attr,omitempty"`
+	Model *Model `xml:"model,omitempty"`
```

Comment:
Do we really need Model? looks like it's not used

Reply:
Gone.